### DISCUSSION TITLE
Is query merge mechanism deleted, Where is the implementation?

### DISCUSSION THREAD

**Discussion by underspirit** [milvus-io/milvus#3950] [Q&A and General discussion]
Milvus supports [query merge mechanism](https://zhuanlan.zhihu.com/p/148274838), and found that #1702 implements and merges this function, but #3022 has a lot of changes, which delete the #1702 implementation file, and no other implementation is found, is this function deleted or implemented somewhere ? @yhmo 
file: core/src/server/delivery/request/SearchCombineRequest.cpp

![image](https://user-images.githubusercontent.com/7677161/95698937-bc9c5100-0c75-11eb-8981-c1dd4d4424dc.png)
![image](https://user-images.githubusercontent.com/7677161/95698946-c4f48c00-0c75-11eb-8a02-38642826903f.png)

**Comment by yhmo** [comment]
Query merge mechanism has been temporarily removed from master/0.11.0 branch.
Please switch to 0.10.x branch.
